Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -174,6 +174,7 @@ class ClearPrivateDataTableViewController: UITableViewController {
              } else {*/
             ClearPrivateDataTableViewController.clearPrivateData(clear).uponQueue(DispatchQueue.main) { _ in
                 // TODO: add API to avoid add/remove
+                // @Brave-devs: Why is this done in the next line? 
```

Comment:
Checking the last change to this line (in 1.6) doesn't reveal much as to why its there. It may be have been due to some legacy related issues. Have you tried removing it and see if there were any issues?

(PS: You could have just commented on the line yourself instead of committing)

Reply:
Tried it still everything worked, so not sure, removing this comment.